src/widgets/refinement-list/refinement-list.js

```diff
@@ -66,18 +70,18 @@ const usage = `Usage:
 refinementList({
   container,
   attribute,
-  [ operator='or' ],
-  [ sortBy = ['isRefined', 'count:desc', 'name:asc'] ],
-  [ limit = 10 ],
-  [ showMore = false],
-  [ showMoreLimit = 10 ],
-  [ cssClasses.{root, noRefinementRoot, searchBox, list, item, selectedItem, label, checkbox, labelText, count, noResults, showMore, disabledShowMore}],
-  [ templates.{item, searchableNoResults, showMoreActive, showMoreInactive} ],
-  [ searchable ],
-  [ searchablePlaceholder ],
-  [ searchableIsAlwaysActive = true ],
-  [ searchableEscapeFacetValues = true ],
-  [ transformItems ],
+  operator?: string = 'or',
+  sortBy?: string[]|function = ['isRefined', 'count:desc', 'name:asc'] ],
```

Comment:
I may be missing the point but it seems to me outputting a short link is a good alternative:
- doc has examples 
- lib size is smaller 
- everything is documented in one single place
I'd love to hear your thoughts on it.

Reply:
Good point – it'd also be easier to maintain.

The only downside of this solution that I see right now is that it wouldn't work offline (not necessary IMO).